www/main/toolbar.js

```diff
@@ -170,6 +173,12 @@ const setListeners = () => {
         },
         bani: payload => loadBani(payload.baniId, payload.verseId, payload.lineCount),
         ceremony: payload => loadCeremony(payload.ceremonyId, payload.verseId, payload.lineCount),
+        settings: payload => {
+          const { settings } = payload;
+          if (settings.action === 'changeFontSize') {
+            global.core.menu.settings.changeFontSize(settings.target, settings.value);
```

Comment:
can you comment on what the payload looks like when it sends settings? Does documentation need to be updated?

Reply:
yes documentation needs to be updated.